python/doc/examples/functional_modeling/vectorial_functions/plot_createMultivariateFunction.py

```diff
@@ -0,0 +1,168 @@
+"""
+=============================
+Create multivariate functions
+=============================
+We can create multivariate functions by different methods. One of the methods is to gather
+multivariate functions. The other one is to create a function having multivariate input using
+tensorization of univariate functions. In this example, we present both methods.
+"""
+# %%
```

Comment:
```suggestion
```

Reply:
I am not sure: in all the other files, there is a # %% that follows the end of the description block.....